src/sentry/issues/escalating.py

```diff
@@ -258,3 +238,39 @@ def is_escalating(group: Group) -> bool:
         )
         return True
     return False
+
+
+def parse_groups_past_counts(response: Sequence[GroupsCountResponse]) -> ParsedGroupsCount:
+    """
+    Return the parsed snuba response for groups past counts to be used in generate_issue_forecast.
+    ParsedGroupCount is of the form {<group_id>: {"intervals": [str], "data": [int]}}.
+
+    `response`: Snuba response for group event counts
+    """
+    group_counts: ParsedGroupsCount = {}
+    group_ids_list = group_counts.keys()
+    for data in response:
+        group_id = data["group_id"]
+        if group_id not in group_ids_list:
+            group_counts[group_id] = {
+                "intervals": [data["hourBucket"]],
+                "data": [data["count()"]],
+            }
+        else:
+            group_counts[group_id]["intervals"].append(data["hourBucket"])
+            group_counts[group_id]["data"].append(data["count()"])
+    return group_counts
+
+
+def _issue_category_dataset(category: GroupCategory) -> Dataset:
+    if category == GroupCategory.ERROR:
+        return Dataset.Events.value
```

Comment:
Should this be `return Dataset.Events`?

Reply:
Notice what we pass to the Snuba query:
<img width="314" alt="image" src="https://user-images.githubusercontent.com/44410/235776585-72e069b5-1a86-42fc-bd63-772e78eee438.png">

Reply:
I think the return type on that method should be `str` then.

Same for https://github.com/getsentry/sentry/pull/48369/files#diff-d6a6dfa40db10fb81a56854fd9c0b315dada63657cefc8c2e8b5c190176153fbR272